apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/StringUtil.java

```diff
@@ -27,6 +27,24 @@ public static boolean isNotEmpty(String str) {
         return !isEmpty(str);
     }
 
+    public static boolean isBlank(CharSequence str) {
+        if (str == null) {
+            return true;
+        }
+
+        for (int i = 0; i < str.length(); i++) {
+            if (!Character.isWhitespace(str.charAt(i))) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public static boolean isNotBlank(CharSequence str) {
+        return !isBlank(str);
+    }
+
```

Comment:
If we don't have to manipulate `CharSequence`, just reuse `org.apache.skywalking.apm.util.StringUtil#isNotEmpty`, only add codes when you really need it